.travis.yml

```diff
@@ -3,5 +3,11 @@ perl:
   - '5.30'
   - '5.28'
   - '5.26'
-install: skip
-script: prove -l
+before_install:
+  - curl -sL --compressed https://git.io/cpm -o ~/bin/cpm
+  - chmod +x ~/bin/cpm
+install:
+  - cpm install -g Dist::Zilla
+  - cpm install -g $(dzil authordeps --missing)
+script:
+  - dzil install
```

Comment:
As discussed, we probably want to change this for `dzil test` (maybe verbose?), since `install` won't show the test result.

Reply:
Done.